charts/quickwit/templates/metastore-deployment.yaml

```diff
@@ -38,6 +38,10 @@ spec:
       serviceAccountName: {{ include "quickwit.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      {{ - with .Values.initContainers }}
```

Comment:
There's an error syntax here, we have to remove the blank between `{{` and ` -` because you'll get this error otherwise:

```shell
$ helm template . --values values.yaml --values values2.yaml
Error: parse error at (quickwit/templates/searcher-statefulset.yaml:46): "-"
```

Here's the overided value file to reproduce the bug:

```
---
configLocation: /tmp/yo.yml
config:
  default_index_root_uri: s3://qw-indexes
  storage:
    s3:
      endpoint: https://fr-par.scw.cloud
      region: fr-par
      access_key_id: SCWXXXXXXX
      secret_access_key: XXXXXXX

initContainers: []

environment:
  QW_METASTORE_URI: s3://qw-indexes

indexer:
  initContainers: &initContainers
    - name: hello
      image: busybox:1.28
      command: ['echo', 'heyyyyy']
    - name: config
      image: busybox:1.28
      command: ['cp', '/tmp/yo.yaml', '/quickwit/node.yaml']

searcher:
  initContainers: *initContainers

janitor:
  initContainers: *initContainers

control_plane:
  initContainers: *initContainers

metastore:
  initContainers: *initContainers
```

Reply:
Also because in your default value file, you defined `initContainers` below each quickwit microservices, the path  seems wrong here:

```
{{ - with .Values.initContainers }}
```

To replace by:

```
{{- with .Values.metastore.initContainers }}
```